### PR TITLE
[CSM] Fix documentation localplayer

### PR DESCRIPTION
**Pull request by Dumbeldor**

### CODE REVIEW

File: doc/client_lua_api.md [hunk 1]
```diff
@@ -692,7 +692,7 @@ Call these functions only at load time!
 * `minetest.get_wielded_item()`
     * Returns the itemstack the local player is holding
 * `minetest.localplayer`
-    * returns LocalPlayer handle. See LocalPlayer below.
+    * Holds a reference to LocalPlayer handle. See LocalPlayer below.
```
Comment (red-001) [inline doc/client_lua_api.md]: object not handle.

File: doc/client_lua_api.md [hunk 2]
```diff
@@ -692,7 +692,7 @@ Call these functions only at load time!
 * `minetest.get_wielded_item()`
     * Returns the itemstack the local player is holding
 * `minetest.localplayer`
-    * returns LocalPlayer handle. See LocalPlayer below.
+    * Holds a reference to LocalPlayer object. See LocalPlayer below.
```
Comment (SmallJoker) [inline doc/client_lua_api.md]: Perhaps similar to `minetest.ui.minimap`?
`Reference to the LocalPlayer object. See `LocalPlayer` class reference for methods.`

Comment (nerzhul) [inline doc/client_lua_api.md]: I agree it's a little bit more better wording in @SmallJoker sentence

**Comment by sofar** [comment]
Please include `:get_name()` documentation if you can, it's also missing.

File: doc/client_lua_api.md (hunk 2 above)
Comment (sofar) [inline doc/client_lua_api.md]: Consistency beats everything. So, if you make it consistent to existing reference text, :+1:

**Comment by bigfoot547** [comment]
I agree @sofar

**Comment by SmallJoker** [comment]
:+1:

**Pull request merged**